src/aui-form-builder/js/aui-form-builder-field-list.js

```diff
@@ -107,6 +107,18 @@ A.FormBuilderFieldList  = A.Base.create('form-builder-field-list', A.Base, [], {
     },
 
     /**
+     * Executed to the last empty column.
+     *
+     * @method _afterAppendInAEmptyColumn
+     * @param {Node} content
+     * @protected
+     */
+    _afterAppendInAEmptyColumn: function(content) {
```

Comment:
Oops, I ended up merging without waiting for the rename, forgot about it :)
Could you send a separate pr just with this in that case?